### PR TITLE
Updating review apps

### PR DESCRIPTION
**Pull request by deadlybutter**
- [ ] Update setup script to run on Heroku post build instead of app.json build
- [ ] Add missing env variables

### CODE REVIEW

**Comment by deadlybutter** [comment]
somehow this worked, yay heroku

**Comment by deadlybutter** [comment]
k looks like everything works, ready for a quick +1

File: app.json [hunk 1]
```diff
@@ -1,10 +1,8 @@
 {
   "name": "phoenix-next",
   "description": ":sparkles:",
-  "scripts": {
```
Comment (weerd) [inline app.json]: Shouldn't we have this for non-Heroku deploys?

Comment (deadlybutter) [inline app.json]: Isnt this a step in envoyer?

Comment (weerd) [inline app.json]: Oh wait, wrong file. Nevermind.

Comment (weerd) [inline app.json]: App.json is unrelated to Envoyer stuff anyhoo.

Comment (deadlybutter) [inline app.json]: yar, good to go then?

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     "build": "npm run clean && npm run modernizr && NODE_ENV=production webpack",
     "clean": "rm -rf public/dist",
     "modernizr": "modernizr -c node_modules/@dosomething/forge/modernizr.json -d public/dist/modernizr.js",
-    "postinstall": "npm run build"
```
Comment (weerd) [inline package.json]: I wonder if this running and also specifying `npm run build` in the deploy hooks on Envoyer is what was causing weird behavior w/ NPM in the past 🤔

**Comment by deadlybutter** [comment]
@DFurnes Sure thing! I only discovered the heroku-postbuild over the weekend on accident

File: package.json (hunk 2 above)
Comment (deadlybutter) [inline package.json]: Yeah I think thats what it was 👍

**Comment by deadlybutter** [comment]
[(1) As a developer I want to update the Heroku environment so I can test more stuff](https://trello.com/c/Af6uLU7K/477-1-as-a-developer-i-want-to-update-the-heroku-environment-so-i-can-test-more-stuff)

**Pull request merged**